### PR TITLE
Rework procedure check status firewallD

### PR DESCRIPTION
**Pull request by dmyar21**
### Description

* Incorrect operation of checking the status of firewalld in the scenarios of a installation and adding a new node. When the firewall status is inactive on the nodes

### Solution

* Adding a new check in the firewallD Status check procedure

### Test Cases
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

**TestCase 1**

Test Configuration:

    Hardware: 4CPU/4GB
    OS: Ubuntu 22.04
    Inventory: AllInOne

Steps:

1. install(firewallD disabled) 

Results:

| Before | After |
| ------ | ------ |
| Procedure fail | Procedure OK |

2. add node(firewallD disabled)

Results:

| Before | After |
| ------ | ------ |
| Procedure fail | Procedure OK |

### Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Integration CI passed
- [ ] Unit tests. If Yes list of new/changed tests with brief description
- [ ] There is no merge conflicts

#### Unit tests
Indicate new or changed unit tests and what they do, if any.

### CODE REVIEW

**Review by ilia1243 — CHANGES_REQUESTED** [review]

File: kubemarine/system.py [hunk 1]
```diff
@@ -380,12 +380,12 @@ def fetch_firewalld_status(group: NodeGroup) -> NodeGroupResult:
...
-    disabled_status = True
+    disabled_status = False
 
     for node_result in list(result.values()):
-        if node_result.return_code != 4 and "disabled" not in node_result.stdout:
-            disabled_status = False
-
+        if node_result.return_code == 4 and "inactive" not in node_result.stdout or \
+                node_result.return_code == 3 and "inactive" in node_result.stdout:
+            disabled_status = True
```
[the remark this answers is not in the document]
Comment (dmyar21) [reply kubemarine/system.py:388]: add new functional, pls review

[the remark this answers is not in the document]
Comment (ilia1243) [reply kubemarine/system.py:387]: This comment is still actual. We cannot sat that the whole group has disabled firewalld if any node has disabled firewalld.

File: kubemarine/system.py [hunk 2]
```diff
@@ -380,12 +380,17 @@ def fetch_firewalld_status(group: NodeGroup) -> NodeGroupResult:
...
     result = fetch_firewalld_status(group)
-    disabled_status = True
+    disabled_status = False
 
     for node_result in list(result.values()):
-        if node_result.return_code != 4 and "disabled" not in node_result.stdout:
+        if node_result.return_code == 4:
+            disabled_status = True
+        elif node_result.return_code == 3 and "disabled" not in node_result.stdout:
+            return disabled_status, result
```
Comment (ilia1243) [inline kubemarine/system.py:389]: Looks like this does not solve the reported problem. The reporter has code == 3 and not have disabled in stdout.
Meanwhile, I still not understand exact TC https://github.com/Netcracker/KubeMarine/pull/284#issuecomment-1315454292 and it is still not described.

**Comment by ilia1243** [comment]
check_paas no longer able to verify firewalld status

**Review by ilia1243 — CHANGES_REQUESTED** [review]

**Review by Imadzuma — COMMENTED** [review]
Are you sure, that this case is reproduced on all-in-one cluster?
I can reproduce this problem on main branch only on several nodes with different firewallD status:
1. not installed
2. installed and enabled
3. installed but disabled
Your fix resolves this exception, but please provide more info, how it can be reproduced in all-in-one case (maybe it's mistake?)

**Comment by ilia1243** [comment]
Reject the MR until Test Case to reproduce the problem is unknown.

**Pull request closed without merging**